components/style/properties/data.py

```diff
@@ -14,6 +14,14 @@
 ALL_SIZES = [(size, False) for size in PHYSICAL_SIZES] + [(size, True) for size in LOGICAL_SIZES]
 
 
+def maybe_moz_logical_alias(product, side, prop):
```

Comment:
The legacy moz-prefixed logical properties are only for inline axis. I guess this can be added to the comment.